deploy_django_app.sh

```
@@ -0,0 +1,51 @@
+}
+
+deploy() {
+        docker build -t notes-app .
+        docker run -d -p 8000:8000 notes-app:latest
+}
+
+echo "******* Deployment Started *******"
+
+if ! code_clone; then
+        echo " the code directory already exist"
+        cd django-notes-app
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)